AY2122S1-CS2103T-T11-2/tp
# Improve help command

### Description

- Modifies UserGuide.md to reflect updated `help` command (closes #15)
- Modifies help window to show a dropdown of brief instructions for each command in LingoGO! (closes #26)

### Type of change

- [ ] CI workflow improvements

### How to test

* Click on the help button on the menu bar or press F1 to bring up the help window
* Click on each of the dropdowns to make sure that each help message is visible
* Preview link: https://tp-clj0azqox-kwokieee.vercel.app/UserGuide.html

### Checklist
